contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3808,6 +3808,7 @@ function_call
     | aggregate_windowed_function                      
     | analytic_windowed_function                       
     | spatial_proc_name_server_database_schema LR_BRACKET function_arg_list? RR_BRACKET 
+    | table_column_name DOT xml_method_call
```

Comment:
We should follow same rule convention as we do for function call and spatial func call. Maybe we can use `xml_proc_name_server_database_schema LR_BRACKET expression_list? RR_BRACKET `, then would even not require xml_method_call explicitly.

Reply:
updated rule to follow the convention